frontend/src/metabase/notifications/pulse/components/ChannelCard.tsx

```diff
@@ -0,0 +1,34 @@
+import cx from "classnames";
+
+import CS from "metabase/css/core/index.css";
+import { Flex, Icon, Paper, Text } from "metabase/ui";
+
+export const ChannelCard = ({
+  onClick,
+  title,
+  channel,
+}: {
+  onClick?: () => void;
+  title: string;
+  channel: "email" | "slack";
+}) => {
+  const iconName = channel === "email" ? "mail" : "slack_colorized";
+
+  return (
+    <Paper
+      withBorder
+      radius="md"
+      p="md"
+      shadow="none"
+      onClick={onClick}
+      className={cx(CS.cursorPointer, CS.bgLightHover, CS.textBrandHover)}
+    >
+      <Flex align="center" gap="sm">
+        <Icon name={iconName} c="brand" />
```

Comment:
The "mail" icon stays blue all the time (according to the design). This **does not affect** the colorized slack icon.

If anyone feels like I should, I can explicitly set the color of the icon to "brand" only when the channel is "email". This seemed "good enough" for me.